src/media/mod.rs

```diff
@@ -208,25 +217,24 @@ impl ::std::fmt::Display for MediaCategory {
     }
 }
 
-/// A builder struct that allows you to build up parameters to a media upload before initiating it.
+/// A builder that provides mechanism to create upload and customize it.
 ///
-/// `UploadBuilder` is your entry point to uploading media to Twitter. This allows you to configure
-/// an upload and set the proper metadata, so that when you `call` it to begin the upload proper,
-/// the resulting [`UploadFuture`] can take care of all the underlying details of the upload itself.
+/// `UploadBuilder` is entry point to uploading media to Twitter.
```

Comment:
Would prefer "`UploadBuilder` is the entry point..." if you're taking out the "your" from the sentence.

Reply:
Ok,